### PR TITLE
[Dungeon] Shadowfang Keep

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1588336378116569500.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` = 3914;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 95, `maxdmg` = 122, `DamageModifier` = 1.01 WHERE `entry` = 3865;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3886;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 112, `maxdmg` = 144, `DamageModifier` = 1.01 WHERE `entry` = 3887;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 4278;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 136, `DamageModifier` = 1.01 WHERE `entry` = 4279;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 136, `DamageModifier` = 1.01 WHERE `entry` = 4274;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 3927;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 4275;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336378116569500.sql:42]: fix newline
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 4275;

```

File: data/sql/updates/pending_db_world/rev_1588336378116569500.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+UPDATE `creature_template` SET `mindmg` = 51, `maxdmg` = 69, `DamageModifier` = 1.03 WHERE `entry` = 3854;
+UPDATE `creature_template` SET `mindmg` = 39, `maxdmg` = 52, `DamageModifier` = 1.03 WHERE `entry` = 5058;
+UPDATE `creature_template` SET `mindmg` = 49, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 3857;
+UPDATE `creature_template` SET `mindmg` = 44, `maxdmg` = 59, `DamageModifier` = 1.03 WHERE `entry` = 3855;
+UPDATE `creature_template` SET `mindmg` = 49, `maxdmg` = 65, `DamageModifier` = 1.03 WHERE `entry` = 3877;
+UPDATE `creature_template` SET `mindmg` = 51, `maxdmg` = 69, `DamageModifier` = 1.03 WHERE `entry` = 3866;
+UPDATE `creature_template` SET `mindmg` = 51, `maxdmg` = 69, `DamageModifier` = 1.03 WHERE `entry` = 3868;
+UPDATE `creature_template` SET `mindmg` = 51, `maxdmg` = 69, `DamageModifier` = 1.03 WHERE `entry` = 3873;
+UPDATE `creature_template` SET `mindmg` = 51, `maxdmg` = 69, `DamageModifier` = 1.03 WHERE `entry` = 3859;
+UPDATE `creature_template` SET `mindmg` = 51, `maxdmg` = 69, `DamageModifier` = 1.03 WHERE `entry` = 2529;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336378116569500.sql:25]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 51, `maxdmg` = 69, `DamageModifier` = 1.03 WHERE `entry` IN (3866, 3868, 3873, 3859, 2529);
```

File: data/sql/updates/pending_db_world/rev_1588336378116569500.sql [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.02 WHERE `entry` = 3872;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` = 3914;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 95, `maxdmg` = 122, `DamageModifier` = 1.01 WHERE `entry` = 3865;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3886;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 112, `maxdmg` = 144, `DamageModifier` = 1.01 WHERE `entry` = 3887;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 4278;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 136, `DamageModifier` = 1.01 WHERE `entry` = 4279;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 136, `DamageModifier` = 1.01 WHERE `entry` = 4274;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336378116569500.sql:40]:
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 136, `DamageModifier` = 1.01 WHERE `entry` IN (4279, 4274);
```

**Comment by Geekena** [comment]
all damage and spells are ok
only last boss is hard to follow because blink

File: data/sql/updates/pending_db_world/rev_1588336378116569500.sql [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` = 3914;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 95, `maxdmg` = 122, `DamageModifier` = 1.01 WHERE `entry` = 3865;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3886;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 112, `maxdmg` = 144, `DamageModifier` = 1.01 WHERE `entry` = 3887;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 4278;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 106, `maxdmg` = 136, `DamageModifier` = 1.01 WHERE `entry` IN (4279, 4274);
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 3927;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 4275;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336378116569500.sql:37]: change to `type_flags`=`type_flags`|4

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1588336378116569500.sql:37]: Please change to what I wrote above

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1588336378116569500.sql]: Done, recheck

Soz cause that, was waiting to hit "Suggestion btn" xD

File: data/sql/updates/pending_db_world/rev_1588336378116569500.sql [hunk 5]
```diff
@@ -27,11 +27,11 @@ UPDATE `creature_template` SET `mindmg` = 22, `maxdmg` = 31, `DamageModifier` =
...
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 85, `maxdmg` = 110, `DamageModifier` = 1.01 WHERE `entry` = 3914;
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 95, `maxdmg` = 122, `DamageModifier` = 1.01 WHERE `entry` = 3865;
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 101, `maxdmg` = 130, `DamageModifier` = 1.01 WHERE `entry` = 3886;
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 112, `maxdmg` = 144, `DamageModifier` = 1.01 WHERE `entry` = 3887;
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 4278;
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 106, `maxdmg` = 136, `DamageModifier` = 1.01 WHERE `entry` IN (4279, 4274);
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.01 WHERE `entry` = 3927;
+UPDATE `creature_template` SET `type_flags = type_flags|4, `mindmg` = 129, `maxdmg` = 166, `DamageModifier` = 1.01 WHERE `entry` = 4275;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588336378116569500.sql:37]: Now some \` was removed :P
Add the missing ones around the "type_flags"

**Comment by Kitzunu** [comment]
Ty for the PR!
